content/blog/2024/06/2024-06-04-jenkins-in-google-summer-of-code-community-bonding-contributors-takeaways.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Jenkins in Google Summer of Code Community Bonding, Contributors’ Takeaways"
+tags:
+- gsoc2024
+- community
+- events
+author: alyssat
+description: "GSoC contributors learning during the bonding period"
+---
+
+The community bonding period is the three weeks between GSoC contributor acceptance and the start of the coding date. This is a vital time for mentors and org-admins to engage with GSoC contributors and set them up for success. This is the time to set expectations and milestones, welcome the contributors to the community, and give them access to the tools they need to succeed during the coding period.
+
+Given that May 26, 2024 was the end of the bonding period, we’ve asked the GSoC contributors what were some of their key takeaways during this period. Here’s what they had to say: 
+
+* *link:https://www.jenkins.io/projects/gsoc/2024/projects/implementing-ui-for-jenkins-infra-statistics/[Implementing UI for Jenkins Infra Statistics]* | contributor:  link:https://www.jenkins.io/blog/authors/shlomomdahan/[Shlomo Dahan]
+
+  1. **Establish a communication schedule:** Set up regular meetings with mentors early on. Knowing how often to meet and discuss progress will help keep you on track and ensure you are receiving timely feedback.
+  2. **Understand the codebase:** Understanding the project’s codebase and infrastructure is crucial. This will help you make informed decisions when you are planning out your roadmap.
```

Comment:
```suggestion
  2. **Understand the codebase:** Understanding the project’s codebase and infrastructure is crucial.
This will help you make informed decisions when you are planning out your roadmap.
```